CarService/CarService.Cqrs/Commands/Handlers/UpdateCarOwnerCommandHandler.cs

```diff
@@ -20,7 +20,7 @@ public UpdateCarOwnerCommandHandler(IUsersService usersService)
             _usersService = usersService;
         }
 
-        public Task<CarOwner> Handle(UpdateCarOwnerCommand request, CancellationToken cancellationToken)
+        public async Task<Unit> Handle(UpdateCarOwnerCommand request, CancellationToken cancellationToken)
```

Comment:
What's Unit? You can use AsyncRequestHandler if you need just Task Handle without generic